hbw/lib/hbw/logger.rb

```diff
@@ -1,7 +1,39 @@
+HBW_FORMAT = ' [HBW] %s'
+
+class HBWFormatterProduction < Logger::Formatter
+  include ActiveSupport::TaggedLogging::Formatter
+
+  def call(severity, time, program_name, message)
+    HBW_FORMAT % super(severity, time, program_name, message.dup.gsub("\\n", '\\'))
```

Comment:
Is it neccessary to use double quotes here? `'\n'` is equivalent to `"\\n"`, isn't it?